src/patternfly/patternfly-drag-drop.scss

```diff
@@ -0,0 +1,63 @@
+.pf-c-draggable {
+  --pf-c-draggable--m-dragging--Cursor: grabbing;
+  --pf-c-draggable--m-dragging--BoxShadow: var(--pf-global--BoxShadow--md);
+  --pf-c-draggable--m-dragging--after--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-draggable--m-dragging--after--BorderColor: var(--pf-global--active-color--100);
+  --pf-c-draggable--m-drag-outside--Cursor: not-allowed;
+  --pf-c-draggable--m-drag-outside--after--BorderColor: var(--pf-global--danger-color--100);
+
```

Comment:
```suggestion

cursor: var(--pf-c-draggable--Cursor);

```

Reply:
Added this but set it to auto per https://github.com/patternfly/patternfly/pull/4398/files#r716069162.